apps/sms/js/thread_list_ui.js

```diff
@@ -393,17 +394,24 @@ var ThreadListUI = {
     }
   },
 
-  markReadUnread: function thlui_markReadUnread(selected, isRead) {
-    selected.forEach((id) => {
+  markReadUnreadUI: function thlui_markReadUnreadUI(selected, isRead) {
+    function filterIds(id) {
       var thread  = Threads.get(id);
       var markable = thread && (!thread.hasDrafts || isRead);
```

Comment:
does this mean we can't mark as "unread" a thread that has a draft ? I don't exactly remember why we did that. Because we thought that if there is a draft it can't be "unread" ?